pybliometrics/scopus/utils/parse_content.py

```diff
@@ -92,9 +94,17 @@ def get_link(dct, idx, path=['coredata', 'link']):
         return None
 
 
-def html_unescape(s:str): 
-        from html import unescape
-        return unescape(s) if s else None
+def get_unescaped_or_raw(item: Dict[str, Any], key: str, un_escape: bool) -> Optional[str]:
+    """Retrieve a value from a dictionary and optionally unescape HTML characters."""
+    value = item.get(key)
+    if value:
+        return html_unescape(value) if un_escape else value
+    return None
```

Comment:
Na, this is not such a good idea. It it quite complex due to its iterative nature. And it doesn't make the code simpler where the function is applied, because it needs three parameters and it is used in another function. It's totally fine when `_join()` becomes a bit more complex, but let's not distribute complexity in multiple layers of function.

Reply:
Sure, I understand. It was mainly meant for [the other attributes](https://github.com/nils-herrmann/pybliometrics/blob/9ce87e08ebccca138c564eb71337d3130a6b36b5/pybliometrics/scopus/scopus_search.py#L87-L112) (title, description, authkeywords).